release/images.sh

```diff
@@ -36,26 +36,49 @@ sign_checksum() {
     fi
 }
 
+finalize_image() {
+    local name_path="${1}"
+    local name="$(basename "${name_path}")"
+
+    local finalize_script="$(find "${TEMPLATES_PATH}" -name "${name}-finalize.*" -print -quit)"
+    if [[ -n "${finalize_script}" ]]; then
+        if ! LOG_NAME="${name}" run_and_log "${finalize_script}" "${name_path}.img"; then
+            log "Image '${name}'" "NOT OK!"
+            return 1
+        fi
+    fi
+
+    local images="$(ls "${name_path}".*)"
+    local final_path="${IMGS_DIR}/${DISTRO_NAME}-${MIX_VERSION}-${name}"
+    for image in ${images}; do
+        local ext="${image#*.}"
+        local final_file="${final_path}"."${ext}"
+        mv "${image}" "${final_file}"
+        sha512sum "${final_file}" > "${final_file}-${CHKSUM_FILE_SUFFIX}"
+        sign_checksum "${final_file}-${CHKSUM_FILE_SUFFIX}"
+    done
+}
+
 create_image() {
     # ${1} - File path to image template file
     local image=${1}
     local name=$(basename "${image%.yaml}")
     local log_file="${LOG_DIR}/clr-installer-${name}.log"
-    local final_file="${IMGS_DIR}/${DISTRO_NAME}-${MIX_VERSION}-${name}"
 
     if [[ -z "${image}" || -z "${name}" ]]; then
         error "Image creation failed. Invalid input" "${1}"
         return 1
     fi
 
     pushd "${WORK_DIR}" > /dev/null
+
+    set +e
```

Comment:
prefer to treat the output of the command over doing this.

Reply:
To be honest, this is no-op. All functions  that are executed via parallel are not affect by the top set -e. So you can just drop this change at all.

Reply:
removed